From b6f0579cfd44a7e5b84b8239663a56f346acfc47 Mon Sep 17 00:00:00 2001
From: Robbie Harwood <rharwood@redhat.com>
Date: Fri, 1 Sep 2017 15:59:49 -0400
Subject: [PATCH] Work around libev not being c89-compliant

(cherry picked from commit c57599d7039d2435c306830b2edae999c462befb)
[rharwood@redhat.com: Remove travis files]
---
 configure.ac | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index 09ee123..b7b5908 100644
--- a/configure.ac
+++ b/configure.ac
@@ -7,8 +7,8 @@ m4_ifdef([AC_USE_SYSTEM_EXTENSIONS],
     [AC_USE_SYSTEM_EXTENSIONS],
     [AC_GNU_SOURCE])
 
-AC_PROG_CC_C89
-for flag in -std=c89 -Wall -Wextra; do
+AC_PROG_CC_C99
+for flag in -Wall -Wextra; do
   AC_TRY_COMPILE([], [return 0;], [CFLAGS="$CFLAGS $flag"],)
 done
 
